diagnostic.md

```diff
@@ -9,6 +9,12 @@ Assume all code executes in strict mode.
 
 Please list the JavaScript primitives and give one example of each.
 
+1.NaN - banana
+2.Boolean - true
+3.Undefined - something that has yet to be defined
+4.String - 'banana'
+5.Number - 5
+
```

Comment:
Missing null and Symbol, but no points off for Symbol since we aren't covering that.